apps/meteor/app/livechat/server/hooks/afterUserActions.ts

```diff
@@ -37,16 +37,8 @@ const handleDeactivateUser = async (user: IUser) => {
 	}
 };
 
-const handleActivateUser = async (user: IUser) => {
-	if (isAgent(user) && user.username) {
-		await LivechatTyped.addAgent(user.username);
```

Comment:
Is losing some of the behaviors inside the addAgent method (e.g setOperator, dispatch some callbacks) the expected result here? I understand the task asks for not to set the User as Available/Not available but what about the other behaviors?

P.S. I'm not saying this is the correct behavior as well, we should not run `livechat.onNewAgentCreated` after changing a user status, but we were already, so might places be relying on that. 😞